### PR TITLE
Update hosting_providers.json

### PR DESCRIPTION
**Pull request by BoryaGames**
I am added hosting 7m.pl

### CODE REVIEW

File: _data/hosting_providers.json [hunk 1]
```diff
@@ -2448,5 +2448,15 @@
...
     "note": "Let's Encrypt available on all hosting, VPS, and dedicated server plans."
+  },
+  {
+    "name": "7m pl",
+    "link": "https://7m.pl",
+    "category": "full",
+    "tutorial": "",
+    "announcement": "",
+    "plan": "",
+    "reviewed": "",
```
Comment (alexzorin) [inline _data/hosting_providers.json:2459]:
```suggestion
    "reviewed": "2020.11.12",
```

**Review by alexzorin — CHANGES_REQUESTED** [review]
Hi @BoryaGames,

Could you please clarify whether the hosting provider provides automatic HTTPS for custom domains as well, not only for `*.7m.pl`? If so, are there any example domains?

**Comment by BoryaGames** [comment]
> Hi @BoryaGames,
> 
> Could you please clarify whether the hosting provider provides automatic HTTPS for custom domains as well, not only for `*.7m.pl`? If so, are there any example domains?

Yes, but i am don't know example domains.

**Comment by alexzorin** [comment]
I wasn't able to verify that HTTPS for custom domains is supported, so I sent the provider an email. I received a response:

>We don't currently offer https for domains, but it will be available soon.

I think the appropriate thing to do here would be to wait for this feature to be implemented.

**Pull request closed without merging**